packages/react-scripts/config/webpack.config.prod.js

```diff
@@ -460,4 +425,9 @@ module.exports = {
     tls: 'empty',
     child_process: 'empty',
   },
+  // Turn off performance hints in production because we utilize
+  // our own hints via the FileSizeReporter
```

Comment:
Is `FileSizeReporter` really used? I cannot find reference in `webpack.config.*.js`

Reply:
> It's called from the build script after webpack is done

cc here, thanks andriijas